src/os/CMakeLists.txt

```diff
@@ -91,7 +91,7 @@ if(WITH_LTTNG)
   add_dependencies(os bluestore-tp)
 endif()
 
-target_link_libraries(os kv)
+target_link_libraries(os $<$<PLATFORM_ID:Windows>:ceph-common> kv)
```

Comment:
if `os` uses symbols exposed by ceph-common, there is no reason to make this Windows specific.

Reply:
I kept those checks as there were a few cases in which the linker was complaining about duplicate symbols when targeting Linux. I didn't want to spend much time on that, but I can drop the checks and see which one causes issues.

Reply:
i think, to get things right takes time.